test/Microsoft.NET.Build.Containers.IntegrationTests/CreateNewImageTests.cs

```diff
@@ -151,6 +151,16 @@ public void Tasks_EndToEnd_With_EnvironmentVariable_Validation()
             .Execute()
             .Should().Pass();
 
+        // Here begins the hack to get the test to work until we get the nightly builds.
```

Comment:
can you use `changeTargetFrameworkAfterAppCreation` here?

Reply:
originally didn't want to call across test suites and there is no "utilities" file for the integration tests. 
That being said, I will do it.